app/views/miq_request/_prov_dialog_volume_fieldset.html.haml

```diff
@@ -17,6 +17,7 @@
 
 .form-horizontal#add-volumes-form
   - while (draw_fields)
+    = options
```

Comment:
I'm trying to understand why is the above line here? Why are we rendering the `options` hash here?

Reply:
Sorry, that's a debug line that got left in. Thanks for noticing it.